commercetools-models/src/test/java/io/sphere/sdk/products/search/ProductProjectionSearchModelIntegrationTest.java

```diff
@@ -32,7 +32,7 @@
     protected static ProductType productType;
 
     private static final boolean expectAggregations = Optional.ofNullable(System.getenv("EXPECT_AGGREGATIONS"))
-            .map("true"::equals).orElse(false);
+            .map("true"::equals).orElse(true);
```

Comment:
Here we say "if this var exists (and does not contain true) it will be false, otherwise by default is true". But it feels a bit weird because usually a var that contains something is `true` except for few cases (`false`, null, 0, empty...).

Of course the code becomes a bit more complex:
```
.map(env -> !env.equals("false")).orElse(true);
```

Well, it's a test anyway.